support/proxy/vwf.example.com/blockly/manager.vwf.yaml

```diff
@@ -27,6 +27,6 @@ properties:
 methods:
   startAllExecution:
   stopAllExecution:
+  stopExecutionForNode:
```

Comment:
It might be better to put a `stopExecution` method on the `blockly/controller.vwf` component instead. It makes more sense. Then we won't need to pass the ID.